ansible_collections/arista/avd/examples/single-dc-l3ls/README.md

```diff
@@ -0,0 +1,831 @@
+
+```yml
+---
+servers:
+  dc1-leaf1-server1:
+    adapters:
+    - type: server
+      server_ports: [ PCI1, PCI2 ]
+      switch_ports: [ Ethernet5, Ethernet5 ]
+      switches: [ dc1-leaf1a, dc1-leaf1b ]
+      vlans: 11-12,21-22
+      native_vlan: 4092
+      mode: trunk
+      spanning_tree_portfast: edge
+      port_channel:
+        description: PortChannel dc1-leaf1-server1
+        mode: active
+
+    - type: ilo
+      server_ports: [ iLO ]
+      switch_ports: [ Ethernet5 ]
+      switches: [ dc1-leaf1c ]
+      vlans: 11
+      mode: access
+      spanning_tree_portfast: edge
+
+  dc1-leaf2-server1:
+    adapters:
+    - type: server
+      server_ports: [ PCI1, PCI2 ]
+      switch_ports: [ Ethernet5, Ethernet5 ]
+      switches: [ dc1-leaf2a, dc1-leaf2b ]
+      vlans: 11-12,21-22
+      native_vlan: 4092
+      mode: trunk
+      spanning_tree_portfast: edge
+      port_channel:
+        description: PortChannel dc1-leaf2-server1
+        mode: active
+
+    - type: ilo
+      server_ports: [ iLO ]
+      switch_ports: [ Ethernet5 ]
+      switches: [ dc1-leaf2c ]
+      vlans: 11
+      mode: access
+      spanning_tree_portfast: edge
+```
+
+This defines the settings for the relevant switch ports to which the endpoints connect, in this case the two servers `dc1-leaf1-server1` and `dc1-leaf2-server1`.
+
+If we look closer at the definitions for `dc1-leaf1-server1`, we see the following:
+
+```yml
+  dc1-leaf1-server1:
+    adapters:
+    - type: server
+      server_ports: [ PCI1, PCI2 ]
+      switch_ports: [ Ethernet5, Ethernet5 ]
+      switches: [ dc1-leaf1a, dc1-leaf1b ]
+      vlans: 11-12,21-22
+      native_vlan: 4092
+      mode: trunk
+      spanning_tree_portfast: edge
+      port_channel:
+        description: PortChannel dc1-leaf1-server1
+        mode: active
+
+    - type: ilo
+      server_ports: [ iLO ]
+      switch_ports: [ Ethernet5 ]
+      switches: [ dc1-leaf1c ]
+      vlans: 11
+      mode: access
+      spanning_tree_portfast: edge
+```
+
+- The relevant `adapters` are defined:
+  - `type` set to `server` and `ilo` is purely for documentation and readability purposes. It has no operational significance.
+  - `server_ports` are defined purely for use in the interface descriptions on the switch. This doesn't configure anything on the server.
+  - `switch_ports` define the interfaces used in the switches. In this example the server is dual-connected to Ethernet5 and Ethernet5. These two ports exist on switch dc1-leaf1a and dc1-leaf1b defined in the following line.
+  - `switches` define the switches used, in this case dc1-leaf1a and dc1-leaf1b. Note that the `server_ports`, `switch_ports` and `switches` definitions are paired vertically.
+  - `vlans` define which VLANs are allowed on the switch_ports, in this case it's two ranges, VLAN11-12 and VLAN21-22 for the dual-attached server ports and VLAN11 for the iLO port.
```

Comment:
I guess it is really `defines` and not `define` given that these are variable names

Reply:
Fixed